installation/docker-compose.adoc

```diff
@@ -4,9 +4,9 @@ include::../partials/attributes.adoc[]
 {CANVAS} and its related services can be run on a single machine using `Docker Compose for BAMOE Canvas`.
 We can access the docker-compose file from the https://www.ibm.com/support/pages/node/7001673[Support and Download Page].
 
-== Prerequistes
+== Prerequisites
 
-You will need Docker and the Compose plugin in order to run the file. Refer to their web page for installation.
+You need the Docker and Compose plugins to run the file. See the web page for installation.
```

Comment:
"the Docker"... feel not natural

Reply:
Ah.
You will need Docker and Docker Compose ... ?

Reply:
Take out the word "plugins"

Reply:
"You'll need Docker and the Compose plugin" could work

Reply:
Or even:
You need Docker and Docker Compose to run the file.

Reply:
works for me